src/accountant.rs

```diff
@@ -81,6 +81,12 @@ impl Accountant {
         true
     }
 
+    // fn forget_signature_with_last_id(&self, last_id: &Hash) -> bool {
```

Comment:
Don't need this. Even if not commented out, it's not right. It should remove `last_id`, not the last item.